broker.go

```diff
@@ -48,14 +48,18 @@ func generateUniqueEmail(baseEmail string, id string) string {
 func (b awsAccountBroker) Services(ctx context.Context) []brokerapi.Service {
 	return []brokerapi.Service{
 		brokerapi.Service{
-			// TODO change to GUID?
-			// https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md#service-objects
-			ID:          "aws-account-broker",
+			ID:          "1d138a29-ac8b-4360-be9b-db50867fee95",
```

Comment:
What's this correspond to? Seems like something we wouldn't want to hard-code.

Reply:
It doesn't correspond to anything, it is a random (i.e. Version 4) GUID.  It was hardcoded before, but was a simple string instead of a GUID as recommended by the Open Broker API specification.  As far as I can tell, there is no requirement for them to relate to anything, just be unique.  If you want it to relate to something, then we should use Version 3 GUIDs, and then we need to decide on a namespace to use for generating the GUIDs.  I think the only limitation imposed by hardcoding this is you couldn't register multiple instances of the broker in the same CF organization/space without modifying the GUID.

Reply:
Or we could use [Google UUID newUUID function](https://godoc.org/github.com/google/uuid#NewUUID)

Reply:
Depends if needs to be the same across restarts, I suppose. If you're going to hard-code, mind leaving a comment along the lines of what you said above?